Gifski/CustomButton.swift

```diff
@@ -122,6 +122,10 @@ extension CGSize {
 			height: height
 		)
 	}
+
+	func rounded(_ rule: FloatingPointRoundingRule = .toNearestOrAwayFromZero) -> CGSize {
+		return CGSize(width: width.rounded(rule), height: height.rounded(rule))
+	}
```

Comment:
Can you put this in `util.swift`? It's not used in CustomButton.

Reply:
Damn didn't notice that the other `rounded` functions were not in utils but in `CustomButton` file 🤦‍♂